SU-HSDO/suhumsci
# fix(shs-5179): adding missing style settings

# READY FOR REVIEW

## Summary
This PR fixes an issue where the vertical card linked image style is missing any settings at mobile sizes.

## Urgency
medium

## Steps to Test
1. Visit a page like: https://southasia.suhumsci.loc/
2. Note that there is now a preset for the smallest width here where there was not before: 
https://southasia.suhumsci.loc/admin/config/media/image-styles/manage/postcard_vertical_linked_465px_

Before: 
![Screenshot from 2024-01-06 13-55-04](https://github.com/SU-HSDO/suhumsci/assets/2343957/61021ac7-02ac-4c0e-afda-1ea3dfd59b27)

Because of the nature of local testing, I'm having trouble ensuring that it is creating new images properly. I flushed the images, but I don't know if it can create new images locally? Right now, for me, they are showing up broken. 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
